### PR TITLE
Fixing style in manage.py to align with pylint

### PR DESCRIPTION
**Pull request by young-k**
Self-explanatory, I fixed the changes by hand with vim scripts and vim

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -128,159 +131,176 @@ def halt(self):
...
   def __init__(self, config):
     self.config = config
-    
+
     if not os.path.exists(self.config.build_dir):
       os.makedirs(self.config.build_dir)
-    
-    self.vagrant_ssh_cmd = '"rm -rf ~/deploy; cp -r /deploy ~; chmod 0400 ~/deploy/.bld/server.pem; python3.5 ~/deploy/manage.py {0} {1}"'
+
+    self.vagrant_ssh_cmd = '"\
```
Comment (kq-li) [inline manage.py]: control character here?

File: manage.py [hunk 2]
```diff
@@ -128,159 +131,176 @@ def halt(self):
...
+        rm -rf ~/deploy;\
+        cp -r /deploy ~;\
+        chmod 0400 ~/deploy/.bld/server.pem;\
+        python3.5 ~/deploy/manage.py {0} {1}\
+        "'
```
Comment (kq-li) [inline manage.py]: and here

File: manage.py [hunk 3]
```diff
@@ -128,159 +131,176 @@ def halt(self):
...
-  print("****The following swarm operations affect PRODUCTION machines****")
-  print("python manage.py swarm lockdown - lock down access to the target production machines")  
-  print("python manage.py swarm join - join the target machines into a Docker Swarm")
-  print("python manage.py swarm configure - upload the Docker Compose file to the first manager of the Swarm")  
-  print("python manage.py swarm clean - remove Docker Stack, containers, and images from all machines")    
-  print("")
-  print("****The following swarm operations affect TESTBED machines****")
-  print("python manage.py swarm-testbed [above command] - applies swarm operation [above command] onto the testbed")    
-  
+  print("""
```
Comment (kq-li) [inline manage.py]: prefer single quotes

Comment (young-k) [reply manage.py]: you can't have multi-line strings with single quotes

Comment (kq-li) [reply manage.py]: `'''<multiline>'''`

File: manage.py [hunk 4]
```diff
@@ -1,335 +1,355 @@
...
+import shutil
+import subprocess
+import sys
+import os
+
+import jinja2
+import texttable
+
+class Config:
+
```
Comment (kq-li) [inline manage.py]: this bothers me a lot honestly plz consider removing

File: manage.py [hunk 5]
```diff
@@ -1,335 +1,355 @@
...
+    self.testbed_dir = os.path.join(self.build_dir, 'testbed')
+    self.testbed_size = int(self.config['Testbed']['TestbedSize'])
+    self.ip_mask_24 = self.config['Testbed']['IPMask24']
+    self.ip_mask_8_offset = int(self.config['Testbed']['IPMask8Offset'])
+
+  def make_table(self):
+    table = texttable.Texttable()
+    table.add_rows(
+        [
```
Comment (kq-li) [inline manage.py]:
```
table.add_rows([
    [...]
])
```
maybe

Comment (kq-li) [reply manage.py]: avoids one extra level of nesting

File: manage.py [hunk 6]
```diff
@@ -1,335 +1,355 @@
...
+        ['ansible-playbook', 'security-lockdown.yml'],
+        cwd=self.config.build_dir,
+        env=temp_env).wait()
+
+  def join(self, is_testbed):
+    if os.name == 'nt':
+      subprocess.Popen(
+          ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format(
+              'swarm-testbed' if is_testbed else 'swarm', 'join')
+          ],
```
Comment (kq-li) [inline manage.py]: plz fix this array format lol wtf

File: manage.py [hunk 7]
```diff
@@ -1,335 +1,355 @@
...
+        ['ansible-playbook', 'swarm-join.yml'],
+        cwd=self.config.build_dir,
+        env=temp_env).wait()
+
+  def configure(self, is_testbed):
+    if os.name == 'nt':
+      subprocess.Popen(
+          ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format(
+              'swarm-testbed' if is_testbed else 'swarm', 'configure')
+          ],
```
Comment (kq-li) [inline manage.py]: and this

File: manage.py [hunk 8]
```diff
@@ -1,335 +1,355 @@
...
+        ['ansible-playbook', 'configure-stack.yml'],
+        cwd=self.config.build_dir,
+        env=temp_env).wait()
+
+  def clean(self, is_testbed):
+    if os.name == 'nt':
+      subprocess.Popen(
+          ['vagrant', 'ssh', '-c', self.vagrant_ssh_cmd.format(
+              'swarm-testbed' if is_testbed else 'swarm', 'clean')
+          ],
```
Comment (kq-li) [inline manage.py]: ^

**Pull request merged**